README.md

```diff
@@ -582,6 +582,8 @@ In this section, we use badges to indicate the targeted Vue version for each plu
 ### Go
 
 - [vite-go](https://github.com/torenware/vite-go) - Integration for Golang web apps.
+- [React](https://docs.gocopper.dev/getting-started/create-project/react) - Integration for [Copper](https://gocopper.dev/).
```

Comment:
I think the main title should be the name of your framework (in this case Copper I think). Also, I'd think it's better to add only one entry instead of two.

Reply:
There are two separate integrations - one for using vite + react, another for vite + react + tailwind. I'm fine with highlighting just one here. Maybe like this?
```
- [Copper + React](https://docs.gocopper.dev/getting-started/create-project/react) - Integration for Copper + React using Vite
```